src/output-json-http2.c

```diff
@@ -153,7 +153,7 @@ static OutputInitResult OutputHttp2LogInitSub(ConfNode *conf, OutputCtx *parent_
     OutputInitResult result = { NULL, false };
     OutputJsonCtx *ojc = parent_ctx->data;
 
-    OutputHttp2Ctx *http2_ctx = SCMalloc(sizeof(OutputHttp2Ctx));
+    OutputHttp2Ctx *http2_ctx = SCCalloc(1, sizeof(OutputHttp2Ctx));
```

Comment:
Curious: I've noticed that here and in some other places you've replaced SCMalloc even though there's no `memset` following. (`output-pgsql`, `output-smtp`), and I've also noticed that there are still SCMalloc's left. Is there a reason for replacing these specific ones, and not others?

Reply:
Good question.

There are some places where a `malloc` is done and then the entire structure is populated (so no zeroing is needed). There are other places where a `calloc` should've been but was using `malloc`

That said, if I've missed anything or was too aggressive, please let me know

Reply:
Thanks for taking the time to explain, I haven't seen any place where the changes seemed out of place, so all good in that sense :)